libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -76,6 +77,7 @@ public MapboxTelemetry(Context context, String accessToken, String userAgent, Ca
     this.schedulerFlusher = schedulerFlusher;
     this.clock = clock;
     this.localBroadcastManager = localBroadcastManager;
+    this.isTelemetryEnabled = TelemetryUtils.retrieveEnabledTelemetry();
```

Comment:
https://github.com/mapbox/mapbox-events-android/pull/46#pullrequestreview-88769615
> We could also add a parameter to the `MapboxTelemetry` (for testing) constructor to make testing easier and remove the necessity of mocking `SharedPreferences`.

In any case, this is not needed right now. That is, removing this line and defaulting to `false` https://github.com/mapbox/mapbox-events-android/pull/46/files#diff-01e79ead4b0d272bdf0972544f4e7692R52 (as you can see we're already doing that) should be enough for now (until we add new tests that need a concrete `isTelemetryEnabled` value).
Try this out and let me know if tests are still passing.